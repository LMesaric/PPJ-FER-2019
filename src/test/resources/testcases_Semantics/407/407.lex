KR_INT 1 int
IDN 1 main
L_ZAGRADA 1 (
KR_VOID 1 void
D_ZAGRADA 1 )
L_VIT_ZAGRADA 1 {
KR_RETURN 2 return
IDN 2 f
L_ZAGRADA 2 (
D_ZAGRADA 2 )
TOCKAZAREZ 2 ;
D_VIT_ZAGRADA 3 }
KR_INT 4 int
IDN 4 f
L_ZAGRADA 4 (
KR_VOID 4 void
D_ZAGRADA 4 )
L_VIT_ZAGRADA 4 {
KR_RETURN 5 return
BROJ 5 0
TOCKAZAREZ 5 ;
D_VIT_ZAGRADA 6 }
