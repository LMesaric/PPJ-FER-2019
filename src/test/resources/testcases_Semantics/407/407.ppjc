int main(void) {
  return f(); // greska
}
int f(void) {
  return 0;
}
