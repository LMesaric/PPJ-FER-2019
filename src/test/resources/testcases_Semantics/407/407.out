<primarni_izraz> ::= IDN(2,f)
